dart/lib/src/protocol/sentry_exception.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:meta/meta.dart';
+
+import 'mechanism.dart';
+import 'sentry_stack_trace.dart';
+
+/// The Exception Interface specifies an exception or error that occurred in a program.
+class SentryException {
+  /// The type of exception
+  String type;
+
+  /// The value of the exception
+  String value;
+
+  /// The optional module, or package which the exception type lives in.
+  String module;
+
+  /// An optional stack trace object
+  SentryStackTrace stacktrace;
+
+  /// An optional object describing the [Mechanism] that created this exception
+  Mechanism mechanism;
+
+  /// Represents a thread id. not available in Dart
+  int threadId;
```

Comment:
its a Long on Java

Reply:
`Long` doesn't exist in Dart, but `int` should do the job https://api.dart.dev/stable/dart-core/int-class.html

Reply:
TIL, thanks